src/app/submission/submission.component.ts

```diff
@@ -420,6 +420,8 @@ export class SubmissionComponent implements OnInit, OnDestroy {
     switch (graphValidationState) {
       case 'Invalid':
         return '#d9534f';
+      case 'Requested':
+        return 'orange';
```

Comment:
We don't need this line AFAIK.

Reply:
Why not?

Reply:
Am I wrong that in Typescript the switch statement also has the  fall-through feature?
So the return is the same for `Requested` and `Pending`.

Reply:
You are not wrong! I thought that was just a java thing, nice :) done